packages/gatsby-theme-docs/src/components/seo.js

```diff
@@ -62,7 +75,9 @@ const SEO = (props) => {
   ].filter(Boolean);
   return (
     <Helmet
-      titleTemplate={`%s | ${siteData.siteMetadata.title} | commercetools`}
+      titleTemplate={`%s | ${siteData.siteMetadata.title} | ${
+        siteContextTitle || 'commercetools'
```

Comment:
We need a fallback because the home-homepage doesn't have a site context. This only works on the microsites.